recipes/notcurses/all/conanfile.py

```diff
@@ -0,0 +1,41 @@
+from conans import ConanFile, CMake, tools
+
+
+class NotcursesConan(ConanFile):
+    name = "notcurses"
+    description = "a blingful TUI/character graphics library"
+    topics = ("graphics", "curses", "tui", "console", "ncurses")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://nick-black.com/dankwiki/index.php/Notcurses"
+    license = "Apache-2.0"
+    exports_sources = ["CMakeLists.txt"]
+    settings = "os", "compiler", "build_type", "arch"
+    requires = "openimageio/2.2.7.0", "libunistring/0.9.10"
+    options = {"shared": [True, False]}
+    default_options = {"shared": True,
+                       "with_doctest": True}
```

Comment:
```suggestion
    default_options = {"shared": True}
```

Also, default shared value will be an issue. CCI asks for static build by default.
And by the way, since your CMakeLists always build shared lib, some patches at build time or cleanup during packaging will be required.

Reply:
> And by the way, since your CMakeLists always build shared lib, some patches at build time or cleanup during packaging will be required.

if we don't want static libs, -DUSE_STATIC=off:

`option(USE_STATIC "Build static libraries (in addition to shared)" ON)`

Reply:
Yes, but if we want static only (shared option False)?

Reply:
I've got this working now, I think. Thanks!